pkg/kubelet/sysctl/runtime.go

```diff
@@ -31,8 +30,13 @@ const (
 	// (e.g., 1.24). Append the version with a ".0" so that it works
 	// with both the CRI and dockertools comparison logic.
 	dockerMinimumAPIVersion = "1.24.0"
+
+	dockerTypeName = "docker"
 )
 
+// TODO: The admission logic in this file is runtime-dependent. It should be
+// changed to be generic and CRI-compatible.
+
```

Comment:
The comment was intended for the entire file, not just the struct below. Hence the extra line for separation.